editions/tw5.com/tiddlers/widgets/ListWidget.tid

```diff
@@ -75,7 +75,7 @@ The content of the `<$list>` widget is an optional template to use for rendering
 |template |The title of a template tiddler for transcluding each tiddler in the list. When no template is specified, the body of the ListWidget serves as the item template. With no body, a simple link to the tiddler is returned. |
 |editTemplate |An alternative template to use for [[DraftTiddlers|DraftMechanism]] in edit mode |
 |variable |The name for a [[variable|Variables]] in which the title of each listed tiddler is stored. Defaults to ''currentTiddler'' |
-|emptyMessage |Message to be displayed when the list is empty |
+|emptyMessage |Message to be displayed when the list is empty. Commands in the widget body are left unevaluated. |
```

Comment:
The trouble with that wording is that it doesn't take into account the `template` attribute, which overrides the widget body. Perhaps it would be clearer to say "Message to be displayed when the list is empty. Note that the message entirely replaces any template specified in the `template` attribute or widget body".